Scripts/tests/emme_only/test_assignment.py

```diff
@@ -73,18 +73,18 @@ def __init__(self):
             project_dir.parent / "Network", scenario_num, "test",
             overwrite=True)
         self.ass_model = ass.EmmeAssignmentModel(emme_context, scenario_num)
-        dist_cost = {
+        self.dist_cost = {
             "car_work": 0.12,
             "car_leisure": 0.12,
             "trailer_truck": 0.5,
             "semi_trailer": 0.4,
             "truck": 0.3,
             "van": 0.2,
         }
```

Comment:
Changed dist cost to instance specific so that we can use it in both trip and freight tests.